### PR TITLE
Configure motor driver signals

### LINKED ISSUE #10: Implement basic motor control with encoder readings
Basically connect microcontroller with motors and driver and make it work (i.e.: go forward and stop, read the traveled distance).

This requires configuring the first PWMs.

### PR DESCRIPTION
**Pull request by Peque**
Fixes #10.

### CODE REVIEW

File: src/main.c [hunk 1]
```diff
@@ -48,6 +48,12 @@ static void setup_gpio(void)
 	gpio_set_mode(GPIOC, GPIO_MODE_OUTPUT_50_MHZ,
 		      GPIO_CNF_OUTPUT_PUSHPULL, GPIO13);
 	gpio_clear(GPIOC, GPIO13);
+
+	/* Motor driver */
+	gpio_set_mode(GPIOB, GPIO_MODE_OUTPUT_50_MHZ,
+		      GPIO_CNF_OUTPUT_PUSHPULL,
+		      GPIO12 | GPIO13 | GPIO14 | GPIO15 );
```
Comment (cua-cua) [inline src/main.c]: Remove space

File: src/main.c [hunk 2]
```diff
@@ -195,23 +201,89 @@ int _write(int file, char *ptr, int len)
...
 	setup_gpio();
 	setup_usart();
 	setup_encoders();
 	setup_pwm();
 	setup_systick();
 
+	drive_forward();
+
 	while(1) {
 		int left = timer_get_counter(TIM1);
```
Comment (cua-cua) [inline src/main.c]: Could you create a function to read the encoders to be more intuitive?

File: src/main.c (hunk 1 above)
Comment (Peque) [reply src/main.c]: Fixed. Added style checks to Travis! :blush:

File: src/main.c (hunk 2 above)
Comment (Peque) [reply src/main.c]: Fixed with 3e276b010bed6f4f976419ac282958231c04595b.

**Pull request merged**